src/wms/src/test/java/org/geoserver/wms/map/RenderedImageMapOutputFormatTest.java

```diff
@@ -1785,6 +1785,21 @@ public void testReprojectionHasNoWhiteLine()
         assertEquals(width * height, validPixelsCount);
     }
 
+    @Test
+    public void testFaultyStyleDoesntBreak() throws Exception {
+        MockHttpServletResponse response =
+                getAsServletResponse(
+                        "wms?BBOX=-180,-90,180,90"
+                                + "&styles=raster&layers=cdf:Fifteen&Format=image/png"
+                                + "&request=GetMap"
+                                + "&width=256"
+                                + "&height=256"
+                                + "&srs=EPSG%3A4326");
+
+        BufferedImage image = ImageIO.read(getBinaryInputStream(response));
```

Comment:
This is supposed to return a blank image, if I understand correctly. If so, assert it using the ``assertBlank`` utility method.

Reply:
done